### PR TITLE
Vijayasree/ Label color

### PR DESCRIPTION
**Pull request by vijayasree-deriv**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/affiliate-date-picker.js [hunk 1]
```diff
@@ -157,6 +155,7 @@ const AffiliateDatePicker = (props) => {
                 label_color={label_color}
                 top_shift={top_shift}
                 is_date_field={is_date_field}
+                la
```
Comment (aswathy-deriv) [inline src/components/elements/affiliate-date-picker.js:158]: Why is this line added?

Comment (vijayasree-deriv) [reply src/components/elements/affiliate-date-picker.js:158]: this is a mistake , i'll remove it

File: src/components/elements/affiliate-date-picker.js [hunk 2]
```diff
@@ -70,9 +70,7 @@ const DayPickerWrapper = styled.div`
             return is_date_field || current_value
                 ? css`
                       transform: translate(-0.6rem, -2.2rem) scale(0.7);
-                      color: var(
-                          --color-${({ error, labelFocusColor }) => (error ? 'red-1' : labelFocusColor)}
-                      );
+                      color: var(--color-${({ error }) => (error ? 'red-1' : 'green')});
```
[the remark this answers is not in the document]
Comment (vijayasree-deriv) [reply src/components/elements/affiliate-date-picker.js:73]: green is a theme color  - var(--color-green), this is the corresponding color used in the codebase.

**Review by aswathy-deriv — APPROVED** [review]

**Review by NikitK-deriv — APPROVED** [review]

**Review by yashim-deriv — APPROVED** [review]

**Review by michio-rework — CHANGES_REQUESTED** [review]
Please resolve the conflicts, thanks thanks :pray:

**Pull request closed without merging**